features/is-analytics-solution-features/org.wso2.carbon.sp.solutions.is.analytics.feature/src/main/resources/p2.inf

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+metaRequirements.0.namespace = org.eclipse.equinox.p2.iu
+metaRequirements.0.name = org.wso2.carbon.extensions.touchpoint
+metaRequirements.0.optional = true
+
+instructions.configure = \
+org.wso2.carbon.extensions.touchpoint.mkdir(path:${installFolder}/../../deployment/);\
```

Comment:
Shouldn't the runtime be taken into account? Is it copying siddhi apps in wso2/worker/deployment/siddhi-files?